standalone/test/unit_tests/data_conversion_layer/unittest_laserscan_conversions.cpp

```diff
@@ -31,101 +32,139 @@ using namespace psen_scan_v2_standalone;
 
 namespace psen_scan_v2_standalone_test
 {
-static data_conversion_layer::monitoring_frame::Message
-createMsg(const util::TenthOfDegree from_theta = util::TenthOfDegree{ 10 },
+static const int64_t DEFAULT_TIMESTAMP{ 4500000 };
+static const int64_t EXPECTED_TIMESTAMP_AFTER_CONVERSION{ 750000 };  // 4500000 - ((45/360) * 30*10^6)
+
+static data_conversion_layer::monitoring_frame::MessageStamped
+createMsg(const int64_t timestamp = DEFAULT_TIMESTAMP,
```

Comment:
shouldn't it be `createStampedMsg()`

Reply:
Maybe, but you already have `Stamped` in the output type. The same holds for `createMsgs()`, `copyMsgWithNewTimestamp()` and `copyMsgWithNewMeasurements()`. I think adding stamped everywhere is a bit too much. What do you think?

Reply:
`const auto frame{ createMsg() };`
you renamed msg with frame and used auto in a lot of tests. If I am new to the project and read the test I would expect frame to be a msg not a stamped one. I think it would get clear pretty fast, but if a `Stmpt` or `Stamped` could save others a few seconds I would change it. The same goes for `createMsgs()`.
Not totaly sure about the other two since the operation is pretty much clear. As a reader you understand what is happening, the only thing here is if you understand what you have to pass in as developer. I'd probably add the stamped here as well.